### ISSUE TITLE
Dist css with sourceMappingURL

### ISSUE THREAD

**Issue by freefri** [mattlewis92/angular-bootstrap-calendar#36]
I have a problem with the minified.css file found in /dist/css/angular-bootstrap-calendar.min.css
This file has the line /*# sourceMappingURL=angular-bootstrap-calendar.min.css.map */
I want to use this file in my production environment, without need to edit it, but if I do it I get a 404 error, because in my server I dont want to provide the .map file.

I dont really understand why is the file angular-bootstrap-calendar.min.css.map needed. Maybe do you added it to Gulp on porpous or is it a default configuration (I'm not really familiar with Gulp)?
Maybe the css task could be just like this:
```
gulp.task('css', function() {

  return gulp.src('src/**/*.css')
    .pipe(gp.sourcemaps.init())
    .pipe(gp.concat('angular-bootstrap-calendar.css'))
    .pipe(gulp.dest('dist/css'))
    .pipe(gp.minifyCss())
    .pipe(gp.rename('angular-bootstrap-calendar.min.css'))
    .pipe(gulp.dest('dist/css'));

});
```

**Comment by mattlewis92** [comment]
You're right, I copied that gulpfile from another project which used less, the sourcemaps aren't needed. FYI though the sourcemaps will only load and 404 when you open the console, so it doesn't really cause any problems other than the error in the console.

**Comment by freefri** [comment]
Yes, it is just an error in the console, but is also one more request and I like to see a clean console. Would be nice if you can fix it and make a new release

**Comment by mattlewis92** [comment]
I'll fix it next time I have an evening free as there's a few other issues to work through as well.

**Comment by mattlewis92** [comment]
Fixed in 0.8.0
